ci/goreleaser/goreleaser.yml

```diff
@@ -104,6 +118,34 @@ dockers:
     - "policies"
     - "coprocess"
     - "tyk.conf.example"
+    
+- ids:
+    - std
+  image_templates:
+    - "tykio/tyk-gateway:{{ .Tag }}-s390x"
```

Comment:
It is better to use the manifest infra under `docker_manifests` section for DX.

Reply:
Does it enough here https://github.com/TykTechnologies/tyk/pull/4777/files#diff-fb944a05459e4d713bc7541efd6e721cbe992a556353c09c4eb66a8eae9b856eR259-R266 ?

I mean I just copied exactly what we had for `arm64` and `amd64`.